content/3_Silver/Sorting_Custom.mdx

```diff
@@ -1176,3 +1178,50 @@ Many of the problems below may use other Silver concepts, such as
 </Info>
 
 <Problems problems="general" hideSuggestProblemButton />
+
+## Quiz
+
+<Quiz>
+  <Quiz.Question>
+    When are custom comparators used?
+    <Quiz.Answer>
+      When sorting integers in default order.
+      <Quiz.Explanation>
+        Incorrect. Sorting integers in default order does not require custom comparators as the default comparator in the sort function suffices.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer>
+      Comparing the relationships of elements in a single-element array.
+      <Quiz.Explanation>
+        Incorrect. Comparators are only needed for data with two or more elements to compare.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+    <Quiz.Answer correct>
+      Sorting custom objects or sorting values in non-default order.
+      <Quiz.Explanation>
+        Correct. You will need to use a custom comparator as there is no default option that is able to sort the custom data, or no option to sort in the specific way that you want.
+      </Quiz.Explanation>
+    </Quiz.Answer>
+  </Quiz.Question>
+  <Quiz.Question>
+    Which input constraints require coordinate compression for the following problem: Given $N$ unique integers, in where each integer is $N_i$ ($1 \leq N_i \leq M$), the solution requires you to create an array where the values of the $N$ integers given are represented by the array's indices.
```

Comment:
idk this problem seems a bit contrived to me, if ur gonna ask a similar question maybe make it a bit more natural

Reply:
Yep sounds good, I will alter the problem.